spec/Builder/ResponseBuilderSpec.php

```diff
@@ -32,4 +32,17 @@ function it_reads_headers_from_string(ResponseInterface $response)
         $this->beConstructedWith($response);
         $this->setHeadersFromString("HTTP/1.1 200 OK\r\nContent-type: text/html\r\n");
     }
+
+    /**
+     * @link https://github.com/php-http/message/issues/41
+     */
+    function it_not_broke_headers(ResponseInterface $response)
```

Comment:
Please do the rename @dbu suggested as it better explains what we exactly test there.